src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.service;
+
+public interface FruitService {
+
+    void toProcessActivities(String fileName);
```

Comment:
If this method wants **to process activities** it must get **Collection of Activities** as an argument.
```suggestion
    void processTransactions(List<FruitTransaction> transactions);
```